VCFv4.3.tex

```diff
@@ -1415,7 +1415,8 @@ \subsection{Header}
 
 The BCF2 header begins with the ``BCF2 magic'' 5 bytes that encode {\tt BCF\em XY} where {\em X} and {\em Y} are bytes indicating the major number (currently 2) and the minor number (currently 2).
 This magic can be used to quickly examine the file to determine that it's a BCF2 file.
-Immediately following the BCF2 magic is the standard VCF header lines in text format, beginning with \verb|##fileformat=VCFvX.Y|.
+Immediately following the BCF2 magic is an unsigned 4-byte integer containing the length of the header text which includes the NUL terminator character.
+Follow the standard NUL-terminated VCF header lines in text format, beginning with \verb|##fileformat=VCFvX.Y|.
```

Comment:
This is now technically correct, describing the length field and NUL-termination that were previously omitted as noted in https://github.com/samtools/hts-specs/issues/236#issuecomment-423999865.

However it's completely incomprehensibly expressed. Either the PR submitter or the VCF maintainers should rewrite this so that it is readable by people who don't already know what the format is.

And similarly in the other two documents.